input/pagecontent/StructureDefinition-ViewDefinition-notes.md

```diff
@@ -21,76 +25,76 @@ All view runners must implement these FHIRPath capabilities:
 * Math operators: [addition (+)](https://hl7.org/fhirpath/#addition), [subtraction (-)](https://hl7.org/fhirpath/#subtraction), [multiplication (*)](https://hl7.org/fhirpath/#multiplication), [division (/)](https://hl7.org/fhirpath/#division)
 * Comparison operators: [equals (=)](https://hl7.org/fhirpath/#equals), [not equals (!=)](https://hl7.org/fhirpath/#not-equals), [greater than (>)](https://hl7.org/fhirpath/#greater-than), [less or equal (<=)](https://hl7.org/fhirpath/#less-or-equal)
 
-<sup>*</sup> Not yet part of the normative FHIRPath release, currently in draft.
+<sup>*</sup> Not yet part of the normative [FHIRPath](https://hl7.org/fhirpath/) release, currently in draft.
+
+#### Optional Core FHIRPath Functions
+
+*View runners* are encouraged to implement these functions as well to support a
+broader set of use cases:
+
+* [memberOf](https://hl7.org/fhir/R4/fhirpath.html#functions) function
+* [toQuantity](https://hl7.org/fhirpath/#toquantityunit-string-quantity) function
 
-#### Additional functions
+#### Required Additional Functions
 
-All view runners must implement these functions that do not exist in the
-FHIRPath specification but are necessary in the context of defining views:
+All *view runners* must implement these functions that extend the
+FHIRPath specification. Despite not being in the [FHIRPath](https://hl7.org/fhirpath/) specification, they are necessary in the context of defining views:
 
-##### getResourceKey() : KeyType
+* [getResourceKey](https://hl7.org/fhir/R4/fhirpath.html#functions) function
+* [getReferenceKey](https://hl7.org/fhirpath/#toquantityunit-string-quantity) function
+
+##### getResourceKey() : *KeyType*
 
 This is invoked at the root of a FHIR [Resource](https://build.fhir.org/resource.html) and returns
-an opaque value to be used as the primary key for the row associated with the resource. In many cases 
+an opaque value to be used as the primary key for the row associated with the resource. In many cases
 the value may just be the resource `id`, but exceptions are described below. This function is used in
-tandem with [getReferenceKey()](#getreferencekeyresource-resourcetypecode--keytype), which returns 
-an equal value from references that point to this resource. 
+tandem with [getReferenceKey()](#getreferencekeyresource-resourcetypecode--keytype), which returns
+an equal value from references that point to this resource.
 
-The returned `<KeyType>` is implementation dependent, but must be a FHIR primitive type that can be used
+The returned *KeyType* is implementation dependent, but must be a FHIR primitive type that can be used
 for efficient joins in the system's underlying data storage. Integers, strings, UUIDs, and other primitive
-types may meet this need.
+types are appropriate.
 
 See the [Resource Keys and Joins](#resource-keys-and-joins) section below for details.
```

Comment:
This link and the link text will need to be updated, as the heading has changed.

Reply:
Good catch. Fixed!